README.md

```diff
@@ -12,12 +12,16 @@ Minecraft Version
 ------------------
 VoxelSniper is being built against the Spigot 1.19 API.
 
-- Minecraft 1.19
-- Minecraft 1.18.2
-- Minecraft 1.17.1
-- Minecraft 1.16.5
+What version do I pick?
 
-***We will not support anything below 1.16.5***
+| Minecraft version | VoxelSniper version                                                                |
+|-------------------|------------------------------------------------------------------------------------|
+| *1.16*\*          | [8.3.0](https://github.com/KevinDaGame/VoxelSniper-Reimagined/releases/tag/v8.3.0) |
+| *1.17*\*          | [8.3.0](https://github.com/KevinDaGame/VoxelSniper-Reimagined/releases/tag/v8.3.0) |
+| 1.18              | [Latest](https://github.com/KevinDaGame/VoxelSniper-Reimagined/releases/latest)    |
+| 1.19              | [Latest](https://github.com/KevinDaGame/VoxelSniper-Reimagined/releases/latest)    |
+
+\* Please note that we will not provide support for the latest voxelsniper version
```

Comment:
```suggestion
\* Please note that we will not provide support when using the latest VoxelSniper version with this Minecraft version
```

Reply:
We leveren uberhaubt geen support voor 8.3.0 meer vind ik. Dat valt niet te onderhouden als er bugs zijn